pennylane/tape/interfaces/autograd.py

```diff
@@ -149,17 +149,73 @@ def get_parameters(self, trainable_only=True, return_arraybox=False):
 
         return params if return_arraybox else autograd.builtins.list(params)
 
-    @autograd.extend.primitive
     def _execute(self, params, device):
-        # unwrap all NumPy scalar arrays to Python literals
-        params = [p.item() if p.shape == tuple() else p for p in params]
-        params = autograd.builtins.tuple(params)

```

Comment:
Don't be confused by how the diff is displayed. These lines were part of actual_execute() and have been moved to lines 215 and following